build/root/WORKSPACE

```diff
@@ -50,8 +50,8 @@ docker_repositories()
 
 http_file(
     name = "kubernetes_cni",
-    sha256 = "05ab3937bc68562e989dc143362ec4d4275262ba9f359338aed720fc914457a5",
-    url = "https://storage.googleapis.com/kubernetes-release/network-plugins/cni-amd64-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz",
+    sha256 = "f04339a21b8edf76d415e7f17b620e63b8f37a76b2f706671587ab6464411f2d",
+    url = "https://storage.googleapis.com/kubernetes-release/network-plugins/cni-plugins-amd64-v0.6.0.tgz",
```

Comment:
Will this get us a sha file as well?

Reply:
@freehan Have you updated the sha file?

Reply:
ah...nope. Why do you need the sha file?

Reply:
ack @chrislovecnm 

I think we only need to know the sha value, which is exactly the same with that of [cni-plugins github releases](https://github.com/containernetworking/plugins/releases), since the tarball stays unchanged, right?

If you have other requirements, @freehan can help upload it.

Reply:
Yes, @freehan should just move over the shasums

Reply:
We have sha files for every other tar package with the k/k release. But no sha file for this file. That is not great consistency with our release. Does bazel support it? Does k/k need it no, but systems that consume cni need it.

Reply:
@chrislovecnm FWIW, I think the GCS thing here is just about storing the exact bins/shasums listed in https://github.com/containernetworking/cni/releases on GCS instead of Github. The shasums should also be moved over

Reply:
Ack. I will upload the sha files